### DISCUSSION TITLE
Provision a cluster without control plane nodes

### DISCUSSION THREAD

**Discussion by therealak12** [kubernetes-sigs/cluster-api#7656] [Q&A] [answered]
Hi.
I was wondering if capi can be used to provision a cluster without control plane nodes. Looking at famous k8s services I see only worker nodes and no control-plane nodes. Is there any way to deploy control-plane components somewhere outside the cluster and have no control-plane nodes?

**Comment by fabriziopandini** [comment]
Having a CP is a mandatory requirement
Also managed Kubernetes offerings are creating one for you, even if you don't see it

**Reply by therealak12** [reply]
Thanks @fabriziopandini 
So let me correct my question, is it possible to hide control-plane nodes from users with the current capi features?

**Reply by sbueringer** [reply, marked as answer]
It depends on your control plane provider.

It is not possible with KubeadmControlPlane. But if you are using the AWS EKS / Azure AKS resources for example they will use the corresponding services from the cloud (and thus have the same behavior as them)

**Reply by therealak12** [reply]
Got it. Thanks.
